LoRaEngine/test/LoRaWan.IntegrationTest/LoRaArduinoSerial.Helper.cs

```diff
@@ -74,8 +74,12 @@ public async Task SetupLora(LoraRegion region)
                     await this.setChannelAsync(0, 868.1F);
                     await this.setChannelAsync(1, 868.3F);
                     await this.setChannelAsync(2, 868.5F);
+                    await this.setChannelAsync(3, 867.1F);
```

Comment:
Is there a reason you chose this additional channels?

Reply:
They were part of the Arduino code, we forgot to add when we started